manifests/00-custom-resource-definition.yaml

```diff
@@ -17,3 +17,187 @@ spec:
+                of the ingress controller.  If unset, defaults are used. See NodePlacement
+                for more details.
+              properties:
+                nodeSelector:
+                  description: 'nodeSelector is the node selector applied to ingress
+                    controller deployments.  If unset, the default is:    beta.kubernetes.io/os:
+                    linux   node-role.kubernetes.io/worker: ''''  If set, the specified
+                    selector is used and replaces the default.'
+                  type: object
+              type: object
+            replicas:
+              description: replicas is the desired number of ingress controller replicas.
+                If unset, defaults to 2.
+              format: int32
+              type: integer
+            routeSelector:
+              description: routeSelector is used to filter the set of Routes serviced
+                by the ingress controller. This is useful for implementing shards.  If
+                unset, the default is no filtering.
+              type: object
+          type: object
+        status:
+          description: status is the most recently observed status of the IngressController.
+          properties:
+            availableReplicas:
+              description: availableReplicas is number of observed available replicas
+                according to the ingress controller deployment.
+              format: int32
+              type: integer
+            conditions:
+              description: 'conditions is a list of conditions and their status.  Available
+                means the ingress controller deployment is available and servicing
+                route and ingress resources (i.e, .status.availableReplicas equals
+                .spec.replicas)  There are additional conditions which indicate the
+                status of other ingress controller features and capabilities.    *
+                LoadBalancerManaged   - True if the following conditions are met:     *
+                The endpoint publishing strategy requires a service load balancer.   -
+                False if any of those conditions are unsatisfied.    * LoadBalancerReady   -
+                True if the following conditions are met:     * A load balancer is
+                managed.     * The load balancer is ready.   - False if any of those
+                conditions are unsatisfied.    * DNSManaged   - True if the following
+                conditions are met:     * The endpoint publishing strategy and platform
+                support DNS.     * The ingress controller domain is set.     * dns.config.openshift.io/cluster
+                configures DNS zones.   - False if any of those conditions are unsatisfied.    *
+                DNSReady   - True if the following conditions are met:     * DNS is
+                managed.     * DNS records have been successfully created.   - False
+                if any of those conditions are unsatisfied.'
```

Comment:
Are the line breaks really lost in this translation? 🤨

Reply:
Yeah, I noticed the generated formatting of descriptions are a bit skewed from the source spec. We can potentially try to fix this upstream. It seems painful to make changes in the api docs to be more generator friendly or manually manage the diffs.